### ISSUE TITLE
Fancybox blinks and not displayed correctly in FF developers edition 61.0b3

### ISSUE THREAD

**Issue by kieliszek85** [fancyapps/fancybox#1930]
If fancybox does not work correctly in FF dev edition, it is very propable that it will not work correctly in standard edition after some time, when changes from dev are committed to standard.

It can be checked on any site with fancybox, for example:
https://fancyapps.com/fancybox/3/

**Comment by kieliszek85** [comment]
It is bug that already exists.
See attached gif. The same thing happens on the fancybox site.
![fancybox](https://user-images.githubusercontent.com/39116427/39804170-a11203b6-5373-11e8-93dc-41695858b733.gif)

**Comment by fancyapps** [comment]
I really like that you call it a bug of fancybox while it works perfectly fine on all major versions of all browsers. So, should you maybe create a bugreport for FF?

**Comment by fancyapps** [comment]
I finally managed to find the reason. The new FF CSS engine has a problem when combining `translate3d` and `mix-blend-mode`. So, if you remove or modify this CSS rule, flickering stops   - 

```CSS
.fancybox-infobar {
    mix-blend-mode: difference;
}
```

I hope they will fix this issue before releasing stable version.

**Comment by kieliszek85** [comment]
Thank you, I will check it.
Yestarday this behaviour has been reported to FF as well.

**Comment by fancyapps** [comment]
I found a simple solution. This can be fixed by adding this CSS rule:

```CSS
.fancybox-container {
    -webkit-transform-style: preserve-3d;
}
```

This will be included in the next commit.

**Comment by kieliszek85** [comment]
Simple solution works as exprected. Thank you!

**Comment by Eldenroot** [comment]
Good catch! Thank you

**Comment by MarkoD** [comment]
This seems to be fixed in the latest FF version - I'm using 61.0b7 and FancyBox v3.3.0 works okay without the CSS workaround.

**Comment by Ezra-Siton-UIX** [comment]
Close this topic :)
